src/rest/runner.py

```diff
@@ -85,21 +90,53 @@ def transcribe(self, transcription_id) -> None:
             return
         self.data_handler.merge_transcript_to_status(transcription_id, response["data"])
 
+    def translate(self, task_id) -> None:
+        """Translates the audio file with the given transcription_id."""
+        transcription = self.data_handler.get_status_file_by_id(task_id)
+
+        # TODO: Move the following in the runner handler
```

Comment:
Keeping it here makes sense imo as it excludes disk read time from the timing data

Reply:
indeed is an argument I can live with